src/test/java/net/snowflake/client/AbstractDriverIT.java

```diff
@@ -385,7 +387,11 @@ public static String getFullPathFileInResource(String fileName) {
     ClassLoader classLoader = AbstractDriverIT.class.getClassLoader();
     URL url = classLoader.getResource(fileName);
     if (url != null) {
-      return url.getFile();
+      try {
+        return Paths.get(url.toURI()).toFile().getAbsolutePath();
```

Comment:
Couldn't we use `toAbsolutePath()` method of `Path` to get the path instead of using the `toFile()` first?

Reply:
I will do that. Thank you!!

Reply:
@sfc-gh-pbulawa Can I resolve this comment?